src/components/style/page.css

```diff
@@ -0,0 +1,136 @@
+.book-left > h4 {
+  color: #919191;
+  font-size: 12px;
+}
+
+.book-right > div > div > h4 {
+  font-size: 22px;
+  position: relative;
+}
+
+.book-right > div > button {
+  border: none;
+  height: 26px;
+  width: 160px;
+  background-color: #0290ff;
+  color: rgb(255, 255, 255);
+  border-radius: 3px;
+  margin-bottom: 20px;
+}
+
+.book-right {
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  justify-items: start;
+}
+
+.book-right > div {
+  width: 70%;
+  text-align: start;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.book-left > h3 {
+  color: rgb(0, 0, 0);
+  text-transform: capitalize;
+  font-size: 22px;
+}
+
+.data > h3 {
+  font-size: 30px;
+}
+
+.book-right > div > div > h3 {
+  color: #919191;
+  text-transform: uppercase;
+  font-size: 15px;
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.bookcard {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.actions {
+  display: flex;
+  gap: 10px;
+}
+
+.btn-special {
+  background: none;
+  border: none;
+  padding: 0;
+  font-family: arial, sans-serif;
+  color: #069;
+  cursor: pointer;
+  font-size: 15px;
+}
+
+.book-left {
+  text-align: left;
+  margin-left: 20px;
+}
+
+.btn-special-middle {
+  background: none;
+  border: none;
+  padding: 0;
+  font-family: arial, sans-serif;
+  color: #069;
+  cursor: pointer;
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+  padding-left: 10px;
+  padding-right: 10px;
+  font-size: 15px;
+}
+
+.book-center {
+  display: -ms-grid;
+  display: grid;
+  width: 40%;
+  border-right: 2px solid #e8e8e8;
+  justify-items: end;
+  align-items: center;
+}
```

Comment:
- [ ] Styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) currently you did a good job but you need to improve it
Original
![bk](https://user-images.githubusercontent.com/44745855/132092257-a215d883-2294-47f2-b598-10cc846f17be.png)

your page
![p](https://user-images.githubusercontent.com/44745855/132092277-0eb2d341-c1b2-4432-9821-15ebaacecaa1.png)

Please work on that!